_templates/sidebar_links.html

```diff
@@ -10,27 +10,23 @@
   <script type="text/javascript">
     nbviewer_random_thumbnail();
   </script>
-	       
-</div> 
+
+</div>
 </div>
 
 <div class="sidebarblock">
 <h3>{{ _('Community') }}</h3>
 
   <ul class="simple">
-    <li><a class="reference external" href="https://gitter.im/ipython/ipython/help">
-            Help Chat Room</a></li>
-    <li><a class="reference external" href="http://stackoverflow.com/questions/tagged/ipython">
```

Comment:
Fair enough, during hangout discussion ML was refered to more than SO. But anyway. BTW @rgbkrk is discourse 1 click deploy on rackspace (like ghost) ?